### ISSUE TITLE
Reading: PolyLM: Learning about Polysemy through Language Modeling

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#172]
## 0. Paper
@inproceedings{ansell-etal-2021-polylm,
    title = "{P}oly{LM}: Learning about Polysemy through Language Modeling",
    author = "Ansell, Alan  and
      Bravo-Marquez, Felipe  and
      Pfahringer, Bernhard",
    booktitle = "Proceedings of the 16th Conference of the European Chapter of the Association for Computational Linguistics: Main Volume",
    month = apr,
    year = "2021",
    address = "Online",
    publisher = "Association for Computational Linguistics",
    url = "https://www.aclweb.org/anthology/2021.eacl-main.45",
    pages = "563--574",
    abstract = "To avoid the {``}meaning conflation deficiency{''} of word embeddings, a number of models have aimed to embed individual word senses. These methods at one time performed well on tasks such as word sense induction (WSI), but they have since been overtaken by task-specific techniques which exploit contextualized embeddings. However, sense embeddings and contextualization need not be mutually exclusive. We introduce PolyLM, a method which formulates the task of learning sense embeddings as a language modeling problem, allowing contextualization techniques to be applied. PolyLM is based on two underlying assumptions about word senses: firstly, that the probability of a word occurring in a given context is equal to the sum of the probabilities of its individual senses occurring; and secondly, that for a given occurrence of a word, one of its senses tends to be much more plausible in the context than the others. We evaluate PolyLM on WSI, showing that it performs considerably better than previous sense embedding techniques, and matches the current state-of-the-art specialized WSI method despite having six times fewer parameters. Code and pre-trained models are available at https://github.com/AlanAnsell/PolyLM.",
}

## 1. What is it?
They propose a PolyLM, which learns sense/sememe embeddings for each word.

## 2. What is amazing compared to previous works?
Their model recognizes each sense of the word (e.g. left).
![スクリーンショット 2021-07-01 17 21 46](https://user-images.githubusercontent.com/45454055/124091453-d21a0b00-da90-11eb-89fb-34e9c6d15b7b.png)

## 3. Where is the key to technologies and techniques?
PolyLM is constructed from three components **to obtain a probability distribution over the entire senses/sememes (number of words × number of senses in each word).**
In this paper, they assign multiple senses to more frequent words.
* Input layer: obtain word representaions
* Disambiguation layer: predict sememe representations from inputs
* Prediction layer: predict a probability distribution

There are three loss functions
* Language Modeling loss (J^{LM}): the mean negative log likelihood of the target tokens occurring
![スクリーンショット 2021-07-01 16 08 57](https://user-images.githubusercontent.com/45454055/124081551-a5f98c80-da86-11eb-82d6-dc560f3442f9.png)

* Distinctness loss (J^D): expect one of these senses to have a high probability, and the rest to have a low probability
![スクリーンショット 2021-07-01 16 15 17](https://user-images.githubusercontent.com/45454055/124082316-89118900-da87-11eb-96d4-0e0eb3d09021.png)

* Match loss (J^M): encourage the output of disambiguation layer (not masked) to be similar to the output of prediction layer (masked) 
![スクリーンショット 2021-07-01 16 15 40](https://user-images.githubusercontent.com/45454055/124082356-962e7800-da87-11eb-9665-5df4e5a82a3a.png)

![スクリーンショット 2021-07-01 16 03 52](https://user-images.githubusercontent.com/45454055/124080978-ef95a780-da85-11eb-81b3-8d0383c40452.png)

## 4. How did evaluate it?
They evaluated their model by Word Sense Induction (WSI) task.
![スクリーンショット 2021-07-01 17 17 59](https://user-images.githubusercontent.com/45454055/124090942-4acc9780-da90-11eb-8c7d-3458a979968d.png)

## 6. Which paper should read next?
